apps/mark-scan/frontend/src/pages/accessible_controller_diagnostic_screen.tsx

```diff
@@ -1,288 +1,192 @@
-      <path d="M90.81 357.38c-2.76 0-5-2.24-5-5v-3c0-2.76 2.24-5 5-5s5 2.24 5 5v3c0 2.76-2.24 5-5 5zM103.58 357.38c-2.76 0-5-2.24-5-5v-14c0-2.76 2.24-5 5-5s5 2.24 5 5v14c0 2.76-2.24 5-5 5zM116.35 357.38c-2.76 0-5-2.24-5-5v-25c0-2.76 2.24-5 5-5s5 2.24 5 5v25c0 2.76-2.24 5-5 5z" />
-      {highlight === 'Headphones' && (
-        <g>
-          <path
-            data-testid="headphones"
-            fill={highlightFill}
-            d="m198.68 369.38-19.34-4.36-13.94-27.72 22.78-18.78 19.86 5.02 11.35 25.14 77.63 76.37-20.72 22.52z"
-          />
-          <path d="M361.3 413.79c-8.41 7.93-8.2 20.89-8.02 32.33.21 13.37-.14 21.71-6.77 24.8-13.67 6.37-22.49-1.93-45.98-26.87-1.86-1.97-3.8-4.03-5.81-6.15l2-2c5.26-5.26 5.26-13.82 0-19.08l-75.29-75.29c-.25-7.08-3.06-14.09-8.46-19.49-5.48-5.48-12.76-8.5-20.51-8.5-7.75 0-15.03 3.02-20.51 8.5s-8.5 12.76-8.5 20.51c0 7.75 3.02 15.03 8.5 20.51 5.65 5.65 13.08 8.48 20.51 8.48.08 0 .17-.01.25-.01l74.64 74.64c2.55 2.55 5.94 3.95 9.54 3.95 3.6 0 6.99-1.4 9.54-3.95l1.2-1.2c1.94 2.05 3.81 4.03 5.61 5.94 17.93 19.03 30.27 32.13 44.88 32.13 3.98 0 8.12-.97 12.59-3.05 12.99-6.05 12.75-20.91 12.55-34.02-.15-9.83-.31-19.99 4.89-24.9 3.43-3.24 9.57-4.55 18.25-3.91l.74-9.97c-11.75-.88-20.2 1.28-25.84 6.6zM179.02 329.1c3.71-3.71 8.57-5.56 13.44-5.56 4.87 0 9.73 1.85 13.44 5.56 7.41 7.41 7.41 19.47 0 26.88s-19.47 7.41-26.88 0-7.41-19.47 0-26.88zm110.63 99.72-10.28 10.28c-1.32 1.32-3.62 1.32-4.94 0l-70.1-70.1c3.14-1.4 6.07-3.38 8.64-5.95 2.81-2.81 4.92-6.05 6.33-9.52l70.35 70.35a3.5 3.5 0 0 1 0 4.94z" />
-        </g>
-      )}
-    </svg>
-  );
-}
+const DIAGNOSTIC_STEPS: DiagnosticStep[] = [
+  {
+    button: 'up',
+    label: 'Up',
+    key: 'ArrowUp',
+  },
+  {
+    button: 'down',
+    label: 'Down',
+    key: 'ArrowDown',
+  },
+  {
+    button: 'left',
+    label: 'Left',
+    key: 'ArrowLeft',
+  },
+  {
+    button: 'right',
+    label: 'Right',
+    key: 'ArrowRight',
+  },
+  {
+    button: 'select',
+    label: 'Select',
+    key: 'Enter',
+  },
+  {
+    button: 'help',
+    label: 'Help',
+    key: 'R',
+  },
+  {
+    button: 'volume-down',
+    label: 'Volume Down',
+    key: '-',
+  },
+  {
+    button: 'volume-up',
+    label: 'Volume Up',
+    key: '=',
+  },
+  {
+    button: 'pause',
+    label: 'Pause',
+    key: 'P',
+  },
+  {
+    button: 'rate-down',
+    label: 'Decrease Rate',
+    key: ',',
+  },
+  {
+    button: 'rate-up',
+    label: 'Increase Rate',
+    key: '.',
```

Comment:
nit: Unlikely that keys will change any time soon, but this current setup technically doesn't guarantee that if the diagnostic is successful, the controller will work to actually interact with the UI -- could expose the key mappings from where they're wired up to UI actions and reference them here, maybe?

Sounds like this PR is supposed to be mostly a stub and maybe this logic will change over time, so feel free to ignore for now.

Reply:
I think it's a useful connection to make. Confirm that this is what you were thinking: 9c2455180?